docs/cudax/accessor_with_properties.rst

```diff
@@ -0,0 +1,137 @@
+.. _libcudacxx-extended-api-accessor-with-properties:
+
+``mdspan`` Accessor with Properties
+===================================
+
+**Accessor with Properties** is a CUDA-specific ``mdspan`` accessor that allows users to exploit advanced memory access capabilities by replacing the ``mdspan`` default behavior.
+
+The accessor allows to specify the following properties:
+
+- **Pointer aliasing policy**, see `Pointer Aliasing Policy`_ for more details
+- **Memory alignment**, see `Memory Alignment`_ for more details
+- **Cache eviction policy**, see `Cache Eviction Policy`_ for more details
+- **Prefetch size**, see `Prefetch Size`_ for more details
+
+Types and Utilities
+-------------------
+
+The **main class** to describe the memory access properties
+
+.. code:: cpp
+
+  template <typename ElementType,
+            typename AliasingPolicy,
+            typename Alignment,
+            typename Eviction,
+            typename Prefetch>
+  class accessor_with_properties;
+
+A **proxy class** to allow distinguishing between load and store operations
+
+.. code:: cpp
+
+  template <typename ElementType,
+            typename Restrict,
+            typename Alignment,
+            typename Eviction,
+            typename Prefetch>
+  class accessor_reference;
+
+A **factory** to simplify the creation of an accessor with properties. It supports a variable number of properties and without enforcing a specific order
+
+.. code:: cpp
+
+  template <typename ElementType, typename... UserProperties>
+  auto make_accessor_with_properties(UserProperties... properties) noexcept;
+
+  template <typename ElementType, typename... UserProperties>
+  auto add_properties(UserProperties... properties) noexcept;
+
+A set of **predefined accessors**:
+
+.. code:: cpp
+
+  cuda::streaming_accessor
+  cuda::cache_all_accessor
+
+Pointer Aliasing Policy
+-----------------------
+
+The *aliasing policy* specifies how pointers are resolved by the compiler when they are used in an expression
+
+- ``aliasing_policy::restrict``: **No aliasing**. Pointers doesn't overlap (**default**)
+- ``aliasing_policy::may_alias``: **May alias**. Pointers may overlap
```

Comment:
```suggestion
- ``aliasing_policy::may_alias``: **May alias**. Pointers may overlap.
```

Reply:
Similar for other points of the format:

`xxx: Yyyy. Zzzz` (needs period at the end)

super tiny nitpick so disregard if you want.